doc/SAI-Proposal-ACL-1.md

```diff
@@ -0,0 +1,362 @@
+1. SAI_SWITCH_ATTR_DEFAULT_INGRESS_ACL<br>
+2. SAI_SWITCH_ATTR_DEFAULT_EGRESS_ACL<br>
+
+## ACL table bind/unbind model and match behavior
+The usage of unique object id allocated for ACL table by the create_acl_table function should be uniformly applied to identify the bind point(s). These bind/unbind points described earlier are logical interfaces defined by: physical ports, LAGs, VLANs, RIFs, tunnels, bridge ports, and switch (global). At any given bind point there can be a set of valid ACl table bound to an interface to filter traffic. Various combinatorial use cases and their behavioral expectations are defined as follows:<br>
+1. One valid ACL table bound to a bind point: Higher priority ACL entry is selected , considering the flow matches that entry and its action set is executed.<br>
+2. ACL table bound to multiple bind points: For a specific flow, there can be multiple valid bind points configured with a one or more valid ACL table(s). These bind points can be cascaded in the pipeline. In such a scenario: any packet modification action to be executed (like SET VLAN, SET IP, etc.) will be executed at a specific stage of the bind point. Non-conflicting actions will be resolved across multiple ACLs derived from different bind points. However, any terminal action like DROP/TRAP will be executed right away, specifically to keep the health of counters, etc. sane.<br>
+
+Figure 1 shows the relationship between an ACL table and various bind points. Examples 1 and 2 in the subsequent sections show how to bind/unbind an ACL table.<br>
+
+![SAI acl design](figures/sai_aclobjs.png "Figure 1: Relationship between ACL table ID and various binding points.")
+__Figure 1: Relationship between ACL table ID and various binding points.__<br>
+
+## ACL group management 
+Grouping of ACL tables was supported in the previous version. This specification enhances the ACL table grouping model and primarily the management of ACL groups - for scalability and flexibility. Two new APIs are introduced in saiacl.h object to manage group ID creation and removal. create_acl_table_group - allocates a unique object ID based ACL group table ID and remove_acl_table_group API removes the unique object ID for recycle. This proposal introduces a new ACL group object. ACL groupid is bound to all the previously discussed bind points that are valid for ACL tables.<br>
```

Comment:
can you update the description for ACL group management? an ACL group is a group of ACL tables that can be applied to the bind points. There are various types of ACL groups which defines the packet matching behavior across the ACL tables within one group. 

Current two group types are defined, sequential and parallel. In sequential mode, each ACL table is assigned with a unique priority within that group. With a packet matches multiple ACL entries within a group, only the one with highest table priority within the group and highest entry priority within the table wins. In parallel mode, all ACL tables are matches and non-conflicting actions are executed.